Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/tools/etcd_helper.go

```diff
@@ -238,7 +239,7 @@ func (h *EtcdHelper) getFromCache(index uint64) (runtime.Object, bool) {
 	if found {
 		// We should not return the object itself to avoid poluting the cache if someone
 		// modifies returned values.
-		objCopy, err := conversion.DeepCopy(obj)
+		objCopy, err := api.Scheme.DeepCopy(obj)
```

Comment:
It should, I opened an upstream to track it https://github.com/GoogleCloudPlatform/kubernetes/pull/9958

----- Original Message -----
> > @@ -238,7 +239,7 @@ func (h *EtcdHelper) getFromCache(index uint64)
> > (runtime.Object, bool) {
> >  	if found {
> >  		// We should not return the object itself to avoid poluting the cache if
> >  		someone
> >  		// modifies returned values.
> > -		objCopy, err := conversion.DeepCopy(obj)
> > +		objCopy, err := api.Scheme.DeepCopy(obj)
> 
> I don't know for certain, but it feels like an EtcdHelper should have a
> scheme as a parameter, not simply assume the global one.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/3278/files#r32659158